front/src/Components/Homepage.js

```diff
@@ -127,7 +107,17 @@ class Homepage extends Component{
                       <h1 className="workout-title">Extreme Pullups</h1>
                       <h2 className="workout-author">David Kang</h2>
                       <p className="workout-description">Work your entire back with this intense but quick 20-minute workout.</p>
+                     
+                      <div className="upvotes">
+                      <h2>
+                        {/* need to import font-awesome library for this (here's the documentation for React: https://fontawesome.com/how-to-use/on-the-web/using-with/react) */}
+                        <i className="fas fa-chevron-up" />
+                        <span className="upvote-count">15</span>
+                        <i className="fas fa-chevron-down" />
+                      </h2>
                     </div>
+                    
+                      </div>
                     </a>
                   </div> 
                   <div className="workout workouts-fade-in-animation">
```

Comment:
I think what we should do here is make an individual component for each workout card on the homepage. Because what's going to end up happening is we're going to need to do: 
`WorkoutReturnJSON.map(workout -> <Workoutcard title={{workout.title}} desc={{workout.desc}} />)`